src/MiddlewareFactory.php

```diff
@@ -121,4 +125,114 @@ private function isArrayDefinition(mixed $definition): bool
 
         return is_subclass_of((string) ($definition['class'] ?? ''), MiddlewareInterface::class);
     }
+
+    /**
+     * @param array{0:class-string, 1:string}|Closure $callable
+     */
+    private function wrap(array|Closure $callable): MiddlewareInterface
```

Comment:
```suggestion
    private function wrapCallable(array|callable $callable): MiddlewareInterface
```

Reply:
Also need test with callable as array `[$object, 'method']`.

Reply:
Currently does not support this.

Reply:
Ok, I'm modified issue: https://github.com/yiisoft/middleware-dispatcher/issues/61